pycaret/__init__.py

```diff
@@ -1 +1,5 @@
 from pycaret.utils import __version__
+
+__all__ = ["show_versions"]
```

Comment:
```suggestion
__all__ = ["show_versions", "__version__"]
```
`__version__` is required as this is PEP standard

Reply:
Thanks for reviewing. I will make this change and push once the other PR has been reviewed and merged. That will make the tests pass as well.